### PR TITLE
allow mods to set zoom limits

### PR DESCRIPTION
**Pull request by DeltaNedas**
added Vars.renderer.{minZoom,maxZoom}.
should be set to old limits by default, need to check.

**when testing, minZoom is the limit for zooming out, maxZoom for zooming in**

### CODE REVIEW

File: core/src/mindustry/core/Renderer.java [hunk 1]
```diff
@@ -302,16 +304,19 @@ public void scaleCamera(float amount){
...
         return camerascale;
     }
 
     public float minScale(){
-        return Scl.scl(1.5f);
+        return Scl.scl(minZoom);
+    }
+
+    public float maxScale(){
+        return Scl.scl(maxZoom);
```
Comment (Anuken) [inline core/src/mindustry/core/Renderer.java:319]: Where did the `Math.round` go?

Comment (DeltaNedas) [reply core/src/mindustry/core/Renderer.java:319]: \x7F

File: core/src/mindustry/core/Renderer.java [hunk 2]
```diff
@@ -302,16 +304,19 @@ public void scaleCamera(float amount){
...
+        targetscale = Mathf.clamp(targetscale, minScale(), maxScale());
     }
 
     public float getDisplayScale(){
         return camerascale;
     }
 
     public float minScale(){
-        return Scl.scl(1.5f);
+        return Math.round(Scl.scl(minZoom));
```
Comment (Anuken) [inline core/src/mindustry/core/Renderer.java:315]: minScale wasn't rounded before.

Comment (DeltaNedas) [reply core/src/mindustry/core/Renderer.java:315]: aaaa

File: core/src/mindustry/core/Renderer.java [hunk 3]
```diff
@@ -30,6 +30,8 @@
     public @Nullable Bloom bloom;
     public FrameBuffer effectBuffer = new FrameBuffer();
     public float laserOpacity = 1f;
+    /** minZoom = zooming out, maxZoom = zooming in */
+    public float minZoom = 1.5f, maxZoom = 1.5f * 6;
```
Comment (Anuken) [inline core/src/mindustry/core/Renderer.java:34]: hold up, why is this 1.5*6 instead of 6

Comment (DeltaNedas) [reply core/src/mindustry/core/Renderer.java:34]: oops i thought it was Scl.scl(1.5f) * 6f

**Pull request merged**